src/formaction.cpp

```diff
@@ -75,8 +75,12 @@ std::string FormAction::prepare_keymap_hint(KeyMapHintEntry* hints)
 	 */
 	std::string keymap_hint;
 	for (int i = 0; hints[i].op != OP_NIL; ++i) {
-		keymap_hint.append(
-			v->get_keys()->getkey(hints[i].op, this->id()));
+		std::string bound_keys = utils::join(v->get_keys()->getkeys(hints[i].op,
+					this->id()), ",");
+		if (bound_keys.empty()) {
+			bound_keys = "<none>";
+		}
+		keymap_hint.append(bound_keys);
```

Comment:
@Minoru: Just so you are aware that this is changed here (easy to revert if you don't want this.

This now shows all keys bound to the operations (in the hints line at the bottom of the window).
Old behaviour was to only show 1 key (or `<none>` if no keys were bound to the operation).